lib/QubitFlatfileImport.class.php

```diff
@@ -645,6 +645,45 @@ private function handleVirtualCols()
     }
   }
 
+  /**
+   * Check array of event data from import, check if this exact event already exists.
+   *
+   * @return bool  True if exists, false if not
+   */
+  public function hasDuplicateEvent($event)
+  {
+    if (!isset($this->object) || !$this->object->id)
```

Comment:
hmm actually I need to check for null on that id property, but I think I can just use isset($this->object->id) instead of the 2 conditions